src/core/imported/rdf/rdf/inc/amdrdf.h

```diff
@@ -26,6 +26,7 @@
 
 #include <cstddef>
 #include <cstring>
+#include <cstdint>
```

Comment:
nitpick: Can you move the include one line up to keep them sorted alphabetically?

Reply:
Whoops, unsaved file!